docs/howto/object-storage/s3/credentials.md

```diff
@@ -45,32 +43,58 @@ need to configure your S3 client with it.
 How exactly you do that depends on your preferred client:
 
 === "aws"
-    Install the [`endpoint`](https://pypi.org/project/awscli-plugin-endpoint/) plugin:
-    ```bash
-    pip install awscli-plugin-endpoint
-    ```
-    Then, enable the plugin in your configuration.
-    Open the `aws` configuration file (normally `~/.aws/config`), and add or modify the `[plugins]` section:
-    ```ini
-    [plugins]
-    endpoint = awscli_plugin_endpoint
-    ```
-    Now, create a new profile, named after your {{brand}} region.
-    Set the credentials and endpoint URLs, using the following commands:
-    ```bash
-    aws configure set \
-      --profile <region> \
-      aws_access_key_id <access-key>
-    aws configure set \
-      --profile <region> \
-      aws_secret_access_key <secret-key>
-    aws configure set \
-      --profile <region> \
-      s3.endpoint_url https://s3-<region>.{{brand_domain}}
-    aws configure set \
-      --profile <region> \
-      s3api.endpoint_url https://s3-<region>.{{brand_domain}}
-    ```
+    The AWS CLI comes in two major versions with different features and configuration methods.
+    AWS CLI v1 is the original version that requires Python and pip for installation.
+
+    1. Configuring AWS CLI v1
+       Install the [`endpoint`](https://pypi.org/project/awscli-plugin-endpoint/) plugin:
+       ```bash
+       pip install awscli-plugin-endpoint
+       ```
+       Then, enable the plugin in your configuration.
+       Open the `aws` configuration file (normally `~/.aws/config`), and add or modify the `[plugins]` section:
+       ```ini
+       [plugins]
+       endpoint = awscli_plugin_endpoint
+       ```
+       Now, create a new profile, named after your {{brand}} region.
+       Set the credentials and endpoint URLs, using the following commands:
+       ```bash
+       aws configure set \
+         --profile <region> \
+         aws_access_key_id <access-key>
+       aws configure set \
+         --profile <region> \
+         aws_secret_access_key <secret-key>
+       aws configure set \
+         --profile <region> \
+         s3.endpoint_url https://s3-<region>.{{brand_domain}}
+       aws configure set \
+         --profile <region> \
+         s3api.endpoint_url https://s3-<region>.{{brand_domain}}
+       ```
+    2. Configuring AWS CLI v2
+       AWS CLI v2 supports custom endpoints natively without plugins.
+       You can configure it using command-line options and configuration file.
+       Create a new profile with credentials:
+       ```bash
+       aws configure set \
+         --profile <region> \
+         aws_access_key_id <access-key>
+       aws configure set \
+         --profile <region> \
+         aws_secret_access_key <secret-key>
+       Then configure the endpoint URL using configuration file.
```

Comment:
Missing closing fence, punctuation error, and missing word:

```suggestion
       ```
       Then configure the endpoint URL, using a configuration file.
```